test-utils-base/src/iosMain/kotlin/mainBackground.kt

```diff
@@ -21,14 +21,13 @@ import com.splendo.kaluga.logging.debug
 import platform.CoreFoundation.CFRunLoopRun
 import kotlin.native.concurrent.TransferMode
 import kotlin.native.concurrent.Worker
-import kotlin.native.concurrent.freeze
 import kotlin.native.internal.test.testLauncherEntryPoint
 import kotlin.system.exitProcess
 
 fun mainBackground(args: Array<String>) {
     debug("using background thread for iOS tests")
     val worker = Worker.start(name = "kaluga-test-background")
-    worker.execute(TransferMode.SAFE, { args.freeze() }) {
+    worker.execute(TransferMode.SAFE, { args }) {
```

Comment:
wish they would update the API as TransferMode does nothing now. 

```suggestion
    worker.execute(TransferMode.SAFE, { args }) { // TransferMode has no effect with the new memory manager in use
```